### PR TITLE
MasterWord: Use GML compiler

### PR DESCRIPTION
**Pull request by toadkarter**
**Contributes to #20518**

This PR is my attempt to port the existing MasterWord application to the new GML compiler. 

If my approach is correct, I would be happy to continue looking at some of the other applications that need to be ported. 

Please do let me know if you would like me to make any changes.

### CODE REVIEW

File: Userland/Games/MasterWord/main.cpp [hunk 1]
```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include "WordGame.h"
 #include "MainWidget.h"
+#include "WordGame.h"
```
Comment (circl-lastname) [inline Userland/Games/MasterWord/main.cpp:8]: I think this commit should be squashed with the previous one

Comment (toadkarter) [reply Userland/Games/MasterWord/main.cpp:8]: Sure thing - is there a way for me to do that on my end? I think there is an option to "Squash Commits and Merge" if the rest of the code is ok to go in, but I'm also happy to close this PR and make a new one with just the one commit if that works better.

Comment (toadkarter) [reply Userland/Games/MasterWord/main.cpp:8]: Thanks, much appreciated! That should be done now, hopefully that did the trick

Comment (circl-lastname) [reply Userland/Games/MasterWord/main.cpp:8]: Yep, that looks fine, you can mark this as resolved

**Review by kleinesfilmroellchen — APPROVED** [review]

**Review by gmta — APPROVED** [review]

**Pull request merged**